hw7/searchTree/tree.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <stdbool.h>
+
+// Struct of dictionary tree, where each node consists of a key, a value, and references to two sons
+typedef struct Tree Tree;
+
+// This function creates empty tree
+Tree* createTree();
+
+// This function deletes tree
+void deleteTree(Tree* tree);
+
+// This function checks if the tree is empty
+bool isEmpty(Tree* tree);
+
+// This function adds value by key
+void addValue(Tree* tree, int key, char* value, bool isWork);
```

Comment:
```suggestion
void addValue(Tree* tree, int key, const char* value, bool isWork);
```
Мы же не хотим его менять внутри?